lib/cli.ts

```diff
@@ -62,9 +63,12 @@ const { _: patterns, ...rest } = yargs
     describe: "A prefix for any import to rewrite to another value.",
   })
   .option("nameFormat", {
-    choices: NAME_FORMATS,
     alias: "n",
+    array: true,
+    choices: NAME_FORMATS,
+    coerce: (arr: NameFormatInput[] | undefined) => arr,
     describe: "The name format that should be used to transform class names.",
+    type: "array",
```

Comment:
the option already has `array: true`, what does `type: "array"` do?

Reply:
Good question 😄  I don't think it's necessary, I'll have a look 👍